packages/Webkul/Checkout/src/Models/Cart.php

```diff
@@ -118,15 +118,16 @@ public function haveStockableItems()
     }
 
     /**
-     * Checks if cart have downloadable items
+     * Checks if cart has downloadable items
      *
      * @return boolean
      */
     public function haveDownloadableItems()
```

Comment:
I would suggest to rename this function to "hasDownloadableItems()" as in the annotations.

Reply:
Done. Thank you :)